README.md

```diff
@@ -1,282 +1,23 @@
 # [![Build Status](https://travis-ci.org/dart-lang/build.svg?branch=master)](https://travis-ci.org/dart-lang/build) [![Coverage Status](https://coveralls.io/repos/dart-lang/build/badge.svg?branch=master)](https://coveralls.io/r/dart-lang/build)
 
-This package provides a way of generating files using Dart code, outside of
-`pub`. These files are generated directly on disk in the current package folder,
-and rebuilds are _incremental_.
+These packages provide utilities for generating Dart code outside of `pub`.
```

Comment:
for the general readme, lets remove `outside of pub`, as you will be able to use the barback package to run builders using pub.